package.json

```diff
@@ -4,13 +4,25 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js"
+    "start": "node src/index.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "format:check": "prettier --check \"**/*.{js,jsx,ts,tsx,json,md}\""
```

Comment:
I think we don't have typescript, maybe remove the ts(x) references

Reply:
i deleted ts and tsx
![image](https://github.com/user-attachments/assets/da9ee9bb-9c6f-45e9-9aa0-b0ae81c0c129)